cutils.c

```diff
@@ -1129,7 +1129,7 @@ void rqsort(void *base, size_t nmemb, size_t size, cmp_f cmp, void *opaque)
 
 /*---- Portable time functions ----*/
 
-#if defined(_MSC_VER)
+#if defined(_MSC_VER) || __MINGW32__
```

Comment:
Why is this necessary? Is it to support the old mingw32 aka not mingw-w64?

Reply:
Probably yea, that's @trufae's commit, here:
https://github.com/radareorg/radare2/commit/86529d7cf63fa399b7e2f59680c553e9dd3edfd3
 and sheesh its failing all the mingw checks, idk why :/

Reply:
Mingw32 is really old and no longer maintained AFAIK, mingw-w64 (which also has a 32bit version, despite the name) has been the de facto MinGW for a while I think.

@trufae ?

Reply:
Can either of you confirm which version of MinGW you're using?

My fear is we'd break mingw32 support since we don't run tests on it.

Reply:
![IMG_20241005_142544](https://github.com/user-attachments/assets/2877a8bb-bc62-4044-b0c3-ff061f7a076a)

Reply:
That's mingw-w64 which we already support, I don't think those changes are necessary.

Any chance you can try to build radare without them?